tools/project_gen/project_gen/project.py

```diff
@@ -315,10 +315,10 @@ def generate_dbs(
 
     print("Generating DS distance db")
     ds_site_source_db_ffp = dbs_dir / DS_SITE_SOURCE_DB_FILENAME
+    empirical_db_scripts_dir = scripts_dir / "db_creation" / "empirical_db"
```

Comment:
Any reasons for this being moved?

Reply:
For generating flt dbs it also uses the `empirical_db_scripts_dir` variable and so if DS was already detected and flt needed generating it would trow an error that it was undefined